package.json

```diff
@@ -65,7 +65,7 @@
         "next-swagger-doc": "^0.4",
         "react": "19.0.0",
         "react-dom": "19.0.0",
-        "si-encryption": "github:safeinsights/encryption",
+        "si-encryption": "github:safeinsights/encryption#2c4d79ac9f5189a46cb637c6ff9325ccb3575c7e",
```

Comment:
lets just keep this as `"si-encryption": "github:safeinsights/encryption` for this pr and on local we can modify the commit to point what we need

Reply:
ok, I can update this after the encryption pr is merged then